docs/book/src/user/quick-start.md

```diff
@@ -219,6 +291,49 @@ spec:
         cloud-provider: aws
 ```
 {{#/tab }}
+{{#tab Docker}}
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Machine
+metadata:
+  name: capi-quickstart-controlplane-0
+  labels:
+    cluster.x-k8s.io/control-plane: "true"
+    cluster.x-k8s.io/cluster-name: "capi-quickstart"
+    set: controlplane
```

Comment:
I'd probably remove the `set` label here, since it isn't necessary

Reply:
Removing, thanks